server/fleetv2_http_api/controllers/login_controller.py

```
@@ -3,7 +3,7 @@
 from typing import Tuple
 from typing import Union
 
-from server.fleetv2_http_api import util
+from fleetv2_http_api import util
```

Ruff (0.8.2)

`fleetv2_http_api.util` imported but unused

Remove unused import: `fleetv2_http_api.util`

(F401)